nekoyume/Assets/_Scripts/State/States.cs

```diff
@@ -86,7 +87,7 @@ public void SetWeeklyArenaState(WeeklyArenaState state)
         /// 최초로 할당하거나 기존과 다른 주소의 에이전트를 할당하면, 모든 아바타 상태를 새롭게 할당된다.
         /// </summary>
         /// <param name="state"></param>
-        public void SetAgentState(AgentState state)
+        public async void SetAgentState(AgentState state)
```

Comment:
`async void` returns no exception even if thrown from its body. Unless it is a special case like a callback, it is better to specify that it returns a `Task`.

```suggestion
        public async Task SetAgentState(AgentState state)
```